api/client.yaml

```diff
@@ -1513,6 +1513,248 @@ paths:
             application/json:
               schema:
                 $ref: 'https://raw.githubusercontent.com/moov-io/base/master/api/common.yaml#/components/schemas/Error'
+  /customers/{customerID}/representatives:
+    post:
+      tags: [Customers]
+      summary: Add customer representative
+      description: Add a Customer representative
+      operationId: addCustomerRepresentative
```

Comment:
Looks like you removed the word "Customer" from all the operation IDs above this point, but included it in these operation IDs. Should we keep them consistent since they're all under a `/customers/{customerID/*` route?